test/common/util.js

```diff
@@ -83,6 +87,16 @@ describe('common/util', function() {
     assert.deepEqual(REQUEST_DEFAULT_CONF, { pool: { maxSockets: Infinity } });
   });
 
+  it('should export an error for module instantiation errors', function() {
+    var missingProjectIdError = new Error([
```

Comment:
It's useful if we really love our error message, and don't want anyone to change it one day without at least having the tests catch it.